src/moa/moa_driver/moa_driver/can_interface_jnano.py

```diff
@@ -0,0 +1,87 @@
+import rcply
+from rcply.node import Node
+
+from moa_msgs.msg import CANStamped
+from moa_msgs.srv import CANSendReq
+
+from .drivers.MCP2515 import MCP2515
+
+
+class CANInterfaceJNano(Node):
+    def __init__(self) -> None:
+        super().__init__("CAN_interface")
+
+        self.can = MCP2515()
+        self.can.Init()
+
+        self.can_read_timer = self.create_timer(
+            0.05,  # TODO: Change to veriable or const
+            self.callback_read_can_data,
+        )
+
+        self.publish_can = self.create_publisher(
+            CANStamped,
+            "/raw_can",
+            10
+        )
+
+        self.service_can = self.create_service(
+            CANSendReq,
+            'publish_can_data',
+            self.callback_publish_can_data
+        )
```

Comment:
change to topic reader, topic is /pub_raw_can, using CANStamped message, 10 is the service type, self.callback_publish_can_data